package.json

```diff
@@ -10,34 +10,33 @@
   },
   "dependencies": {
     "babel": "^5.8.29",
-    "chai": "^3.2.0",
```

Comment:
Updates almost all of the dependencies. babel still needs to be updated to the 6.x line but the changes are quite big so I will do that on another friday